js/index.js

```diff
@@ -1,7 +1,118 @@
+// - Yo, the navigator goes first definitely.
+// - What?! You both have the same name?
+
+let hacker1Big = false;
+let hacker2Big = false;
+
+for (i = 0; (i < hacker1.length || i < hacker2.length); i++){
+ if (hacker1.charCodeAt(i) < hacker2.charCodeAt(i)) {
+   hacker2Big = true;
+   break;
+  } else if (hacker1.charCodeAt(i) > hacker2.charCodeAt(i)){
+    hacker1Big = true;
+    break;
+    } else if (hacker1.charCodeAt(i) === hacker2.charCodeAt(i)){ 
+     continue;
+     }
+}
+ switch (true){
+   case hacker1Big :
+       console.log("Yo, the navigator goes first definitely.");
+       break;
+   case hacker2Big :
+       console.log("The driver's name goes first.");
+       break;
+   case (hacker1.length < hacker2.length) :
+       console.log("The driver's name goes first.");
+       break;
+   case (hacker1.length > hacker2.length) :
+       console.log("Yo, the navigator goes first definitely.");
+       break;
+   case (hacker1.length === hacker2.length) :
+       console.log("What?! You both have the same name?");
+       break;
+ }
+
+
+console.log("\nBonus 1 : ");
+
+let para1= "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin tincidunt ligula vel est laoreet, eget tincidunt urna ultricies. Nulla sit amet lacinia eros. Morbi tincidunt urna velit. Nulla faucibus metus at arcu scelerisque mollis. Aliquam erat volutpat. Proin eleifend felis et mi pharetra efficitur. Ut orci ipsum, semper vel fringilla eu, feugiat sit amet urna. Maecenas quis arcu vitae purus rhoncus auctor. Sed nibh lectus, vehicula ac tellus vel, tincidunt ornare ligula. Proin sapien augue, dignissim ac ipsum vel, ultrices hendrerit neque. Nam tristique lectus in ipsum accumsan, eu porta odio posuere. Nunc tincidunt placerat sapien vel dictum. Fusce metus ipsum, euismod at tellus non, condimentum aliquet nisi. Suspendisse nisl elit, porttitor nec tellus nec, sollicitudin egestas leo. Nulla facilisi. Phasellus congue sapien diam, vitae vulputate metus varius eu. Maecenas nulla dui, pretium ut venenatis quis, mattis quis erat. Cras at mauris posuere, pellentesque mi consequat, vehicula lectus. Vivamus nec mi mauris. Sed viverra pulvinar ante eget varius. Integer at scelerisque justo, et hendrerit metus. Donec lobortis, sem non tempus ultricies, felis neque tincidunt est, vel pulvinar massa sapien ac eros. Mauris eu mi varius, dignissim massa vitae, tincidunt velit. Nulla neque nisl, dignissim ut consequat sit amet, dictum at neque. Nam blandit sapien odio, eu dictum dui molestie quis. Ut consectetur lacus a libero scelerisque sollicitudin. Suspendisse molestie, ligula et imperdiet imperdiet, nulla ipsum pulvinar metus, vitae venenatis massa ligula ut lacus. Nullam ac vulputate lectus, accumsan venenatis elit. Sed at vulputate ipsum."
+
+console.log("No of words in the paragraph is : " + para1.split(" ").length);
+
+console.log("\nBonus 2 : ");
+let phraseToCheck = "Amor, Roma";
+let num2 = phraseToCheck.length - 1;
+let num1;
```

Comment:
Well done, Your code is really good. 🚀
The only point that I would consider is to give better names for your variables, just to improve legibility but it is already really awesome.